indico/modules/rb/models/reservations.py

```diff
@@ -349,7 +349,7 @@ def create_from_data(cls, room, data, user, prebook=None, ignore_admin=False):
         reservation.booked_for_name = reservation.booked_for_user.full_name
         reservation.state = ReservationState.pending if prebook else ReservationState.accepted
         reservation.created_by_user = user
-        reservation.create_occurrences(True)
+        reservation.create_occurrences(True, allow_admin=(not ignore_admin))
```

Comment:
Completely unrelated, but I just noticed that `ignore_admin` is missing in the docstring, perhaps we could add it with these changes?